src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = " _           _        \n"
+                + "| |    _   _| | _____ \n"
+                + "| |   | | | | |/ / _ \\\n"
+                + "| |___| |_| |   <  __/\n"
+                + "|_____|\\__,_|_|\\_\\___|\n";
+        System.out.println("Hello! I'm\n" + logo);
+        System.out.println("What can I do for you?");
+
+        Task[] theList = new Task[100];
+        Scanner userInput = new Scanner(System.in);
+        int counter = 0;
+
+        String echo = userInput.nextLine();
+
+        while (!echo.equals("bye")) {
```

Comment:
Consider declaring static final constants for each command (e.g. bye, list, mark) so that they can be easily referenced and updated for future use
```suggestion
private static final String BYE_COMMAND = "bye";
        while (!echo.equals("bye")) {
```